routes/api/stories.js

```diff
@@ -0,0 +1,50 @@
+//2
+const express = require('express');
+
+
+const {asyncHandler} = require('../utils');
+const { PostLike } = require("../../db/models");
+const { restoreUser } = require('../../auth');
+const createError = require('http-errors');
+
+
+
+const router = express.Router();
+
+//Like a story
+router.post('/stories/:id/likes', restoreUser, asyncHandler(async(req, res, next) => {
+    if (!res.locals.authenticated) {
+       return next(createError(401)); 
+    }
+    const { userId, postId } = req.body;
```

Comment:
This is unused and can be deleted

Reply:
The contents of line 19 have been deleted.